src/python/zquantum/optimizers/cma_es_optimizer.py

```diff
@@ -32,13 +33,13 @@ def __init__(self, options):
                 "CMA-ES always keeps track of the history, regardless of the keep_value_history flag."
             )
 
-    def minimize(self, cost_function, initial_params):
+    def minimize(self, cost_function, initial_params: np.ndarray) -> OptimizeResult:
```

Comment:
Do we want to add typing here?

Reply:
I think the typing should be the same as in "grid_search" for example?

Reply:
I made it the same as in `grid_search`. The typing might need sone additional refinement, but it seems good to me for the time being.